.travis.yml

```diff
@@ -10,10 +9,18 @@ python:
 matrix:
   fast_finish: true
 
+env:
+  matrix:
```

Comment:
This allows to define a matrix of env vars in order to test multiple Ansible versions in parallel.
During the Travis CI build the ANSIBLE env var will be assigned the values listed below thus selecting the corresponding tox env listed in tox.ini.

Reply:
So if I understand it correctly, if I set the `matrix` here to contain __only__ `ANSIBLE=2.6` (once Ansible 2.6 is released), nothing will be tested until I define `2.6: ansible26` in the `travis:env`. And If I set  the `matrix` here to contain __only__ `ANSIBLE=2.4`, only Ansible 2.4 will be tested. Is that correct?

Reply:
Exactly.

Note that if you set ``matrix`` to contain **only** ANSIBLE=2.6 without defining a corresponding ``2.6: ansible26`` mapping in ``travis:env``, I suppose ``tox`` will error because it doesn't know about environment ``py*-ansible{26}``. Onestly I've never tried to feed an env var without a corresponding tox env.

Maybe https://tox-travis.readthedocs.io/en/stable/envlist.html can help clarify or we can try to deliberately break it. :grinning: